pkgs/development/ocaml-modules/uring/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildDunePackage
+, cstruct
+, dune-configurator
+, fmt
+, optint
+, mdx
+}:
+
+buildDunePackage rec {
+  pname = "uring";
+  version = "0.5";
+
+  minimalOCamlVersion = "4.12";
+  duneVersion = "3";
+
+  src = builtins.fetchurl {
```

Comment:
This is very surprising. Why not using `fetchurl` from `nixpkgs`?

Reply:
I have no strong feelings

Reply:
The CI bot has strong feelings:

>  error: access to URI 'https://github.com/ocaml-multicore/ocaml-uring/releases/download/v0.5/uring-0.5.tbz' is forbidden in restricted mode

Reply:
Ha. I didn't mean it like that. I _meant_ to make this change immediately, but had to catch folks for lunch and forgot. Updated.